index.php

```diff
@@ -202,6 +202,7 @@
             <?php if ($auth->isAdmin()): ?>
               <a class="menulink" href="javascript:void(0);" onclick="addUser()"><?= $lang["adduser"] ?></a>
               <a class="menulink" href="javascript:void(0);" onclick="editUser()"><?= $lang["edituser"] ?></a>
+              <a class="menulink" href="javascript:void(0);" onclick="loadLastPositionAllUsers()"><?= $lang["lastpositionallusers"] ?></a>
```

Comment:
This should also be available option for all users in case of public tracks. Only for admin user in case of private tracks.

I think we should implement it in such a way: when user checks "Only latest position" checkbox (and it is admin user or tracks are public) we add "all users" keyword to "select user" dropdown list. The keyword must be removed on unchecking the checkbox.
In such case this link is not needed.

Reply:
Ok. It could be done on a separate javascript file that would be loaded when it is admin user or tracks are public.

Reply:
My idea was to modify [toggleLatest()](https://github.com/bfabiszewski/ulogger-server/blob/e842265e5d015480610438880b0c0646f1f1185a/js/main.js#L369) function.
You could just check for admin user ([like here](https://github.com/bfabiszewski/ulogger-server/blob/e842265e5d015480610438880b0c0646f1f1185a/js/track.js#L22)) in javascript function and act accordingly.
Or maybe even better, instead of checking for admin user and public tracks we could just check if there is more than one item in "Select user" list. If there are multiple users listed there we could add "All users" item (on checking "Only latest")?